README.md

```diff
@@ -163,6 +164,14 @@ COMMAND SUPPLIED
 
 `go get github.com/gruntwork-io/git-xargs@v0.0.5`
 
+
+### Installation option 3: Homebrew
+
+If you are [Homebrew](https://brew.sh/) user, you can install by running
+
+```bash
+$ brew install git-xargs
+```
```

Comment:
This is a much nicer install option, so would you mind making this option 1, at the top of the list?

Reply:
Oh, definitely. :)

Reply:
Done with the update.